caluma/caluma_form/tests/test_minio.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import urllib3
+
+from .. import storage_clients
+
+
+@pytest.mark.parametrize(
+    "disable_cert_checks, debug",
+    [(False, False), (False, True), (True, False), (True, True)],
+)
+def test_minio_disable_cert_checks(db, settings, disable_cert_checks, debug):
+    settings.MINIO_DISABLE_CERT_CHECKS = disable_cert_checks
+    settings.DEBUG = debug
+
+    client = storage_clients.Minio()
+
+    # Some hackery to avoid the test taking too long
+    client.client._http.connection_pool_kw["retries"].total = 0
+
+    if debug and disable_cert_checks:
+        assert client.client._http.connection_pool_kw["cert_reqs"] == "CERT_NONE"
+
+        # This should succeed, as we're not verifying the server certificate
+        client.client._http.urlopen("get", "https://self-signed.badssl.com/")
+    else:
+        assert client.client._http.connection_pool_kw["cert_reqs"] == "CERT_REQUIRED"
+
+        with pytest.raises(urllib3.exceptions.RequestError):
+            # This should fail, as we are in fact verifying certificates
+            client.client._http.urlopen("get", "https://self-signed.badssl.com/")
```

Comment:
I understand that faking all this would be very cumbersome but I generally dislike calling real URLs during tests... Seems like a open gate for flaky CI behaviour and the likes. But since I don't have a better idea either we have to live with it I guess.

Reply:
I do agree, but I wanted to test not only whether the setting is there, but also whether it's actually doing (or not doing, in our case) it's thing. I think it'd be obvious enough if it ever begins flaking, and we can remove the line